lua/layers/mode.lua

```diff
@@ -55,9 +56,11 @@ end
 --- thus not recommended.
 ---
 ---@toc_entry   Layers.mode.new()
+---@param name string mode name, will be displayed in the help window
```

Comment:
```suggestion
---@param name string? mode name, will override the configured help window title
```

As the param is optional, we should mark it as such. I'd also prefer a slightly reworded doc string, as this kinda contrasts the configured window title.